examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx

```diff
@@ -0,0 +1,321 @@
+"use client";
+
+import * as React from "react";
+
+import { Participant, Room } from "../../types";
+import { Frame, Json, OnlineUserInfo } from "@mirohq/websdk-types";
+import {
+  useBreakout,
+  useOnlineUsers,
+  useSelectedItems,
+  useTimer,
+} from "../../hooks";
+import { convertTime, formatDisplayTime, isUser } from "../../utils";
+import { RoomConfig } from "../RoomConfig/RoomConfig";
+import { DEFAULT_TIME, Timer } from "../Timer/Timer";
+import {
+  DropdownMenu,
+  IconButton,
+  IconPlus,
+  IconHandFilled,
+  IconDotsThreeVertical,
+  Button,
+} from "@mirohq/design-system";
```

Comment:
We can't use the design system in examples since its not open source =(

Reply:
We've discussed this with @dfosco and given that this example is a bit different (more elaborate), we need some components from there.

Reply:
The problem is that if people clone the example, they will not be able to install it outside of our VPN right?

Reply:
they can install the whole design system, it's public in the NPMJS, the documentation is not public though. It was public at some point but then the team reverted.

Reply:
https://www.npmjs.com/package/@mirohq/design-system

Reply:
Ah cool/weird! thanks for clearing that up. Should we mention something about that in the readme?

Reply:
Sure, let me add something there.

Reply:
Added a note and updated the demo video, just noticed it was showing the address bar pointing to develop.miro.